microceph/ceph/bootstrap.go

```diff
@@ -5,17 +5,17 @@ import (
 	"context"
 	"database/sql"
 	"fmt"
+	"github.com/canonical/microceph/microceph/common"
 	"os"
 	"path/filepath"
 
 	"github.com/canonical/microceph/microceph/database"
 
-	"github.com/canonical/microcluster/state"
 	"github.com/pborman/uuid"
```

Comment:
Hmm, we've been using the LXD import convention thus far.

Basically, splitting the imports into up to 3 groups.

```
import (
    "built-in/packages"

    "external packages (other git repo)"

    "internal packages (same git repo)"
)
```

So in this case it should be something like:
```
import (
	"context"
	"database/sql"
	"fmt"
	"os"
	"path/filepath"

	"github.com/pborman/uuid"

	"github.com/canonical/microceph/microceph/common"
	"github.com/canonical/microceph/microceph/database"
```

(I realize that this particular file was already not fully compliant)